baseline/OSPS-DO.yaml

```
@@ -201,7 +202,10 @@ controls:
         recommendation: |
           In order to communicate the scope and duration of support for the
           project's released software assets, the project should have a
-          SUPPORT.md or an OpenEoX file in a well known location.
+          SUPPORT.md or other documentation explaining the support lifecycle,
```

Did you want to include the suggestion about using `SECURITY.md` that you had in #252 ?

```suggestion
          SUPPORT.md file, a "Support" section in SECURITY.md, or
          other documentation explaining the support lifecycle,
```